### ISSUE TITLE
Different autoplay speed for every event

### ISSUE THREAD

**Issue by knevagi** [yCodeTech/horizontal-timeline-2.0#16]
Is it possible to have a different autoplay speed for every event in the timeline?

**Comment by yCodeTech** [comment]
Could you give me an insight into how this would be used and why, please. Just so I can picture it in my head.

**Comment by knevagi** [comment]
Like in my case I want to display certain dom elements on my screen for each event. For the first event, I will display 5 elements and for the next event, I will display 10 elements based on what the event actually is. So for this purpose, I will require the autoplay speed to vary for each event based on the number of elements I want to display. So is this something I can achieve with this plugin?

**Comment by yCodeTech** [comment]
Ok thanks. It's not currently possible. I'll see if I can sort something out soon. Though I'm not sure how soon that'll be. So make sure that you're "watching" the repo, to get updates.

**Comment by knevagi** [comment]
ok thanks

**Comment by yCodeTech** [comment]
Apologies for taking so long to look into this, this project was put on the back burner due to personal life. Anyway, I have devised a way to implement this feature. It works by adding a key:value pair to the `data-horizontal-timeline` attribute on the specific event content. So this would be like so:

`<li data-horizontal-timeline='{"date": "16/01/2015", "speed": 4}'>`

This would then override the speed in the plugin settings to allow the newer speed for that particular event. Because this option is specific to the event it is added to, it can be used in conjunction to the speed that is set in the settings; ie. on all events where there isn't a speed option inside the data attribute, it uses the speed setting...

```
<li data-horizontal-timeline='{"date": "16/01/2014", "speed": 10}'></li>
<li data-horizontal-timeline='{"date": "16/01/2015", "speed": 4}'></li>
<li data-horizontal-timeline='{"date": "23/10/2017"}'></li>
<li data-horizontal-timeline='{"date": "23/10/2018"}'></li>
```
In the example above, the last 2 events will be the speed as set in the plugin settings (default 8 secs).

This feature will be available in the next update (v2.0.5.3) which I intend to release hopefully within the next week depending on other new feature implementation.

Thanks for your patience and watch out for the upcoming release!
